customer_domains.php

```diff
@@ -386,6 +388,39 @@
 				$domainip = $result_ipandport['ip'];
 				$result = \Froxlor\PhpHelper::htmlentitiesArray($result);
 
+				$pubkey = $result['dkim_pubkey'];
```

Comment:
same as https://github.com/Froxlor/Froxlor/pull/708/files#diff-572ecbf5b225eef929a4814a4281c7aaf00fa9bc439a6e689a1d7af7ccc55e27R616

Reply:
I do not have the time to dig into this...